src/views/SignIn.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+    <div class="row">
+  <div class="signIn w-100">
```

Comment:
Kann hier auch wieder VueBootstrap verwendet werden?
Kann sein, dass sich das normale BT und das VueBT irgendwann in die Haare bekommen 🤷‍♂️

```suggestion
  <b-row>
    <b-col class="signIn">
```

Reply:
Hat eine b-col automatisch eine weite von 100%?